extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -172,47 +234,61 @@ export const TransactionDetails = ({ goBack }: { goBack: () => void }) => {
     })();
   }, [destAsset.code, destAsset.issuer, networkDetails]);
 
-  const computeDestMinWithSlippage = (
-    slippage: string,
-    destMin: string,
-  ): BigNumber => {
-    const mult = 1 - parseFloat(slippage) / 100;
-    return new BigNumber(destMin).times(new BigNumber(mult));
-  };
+  useEffect(() => {
+    dispatch(getBlockedAccounts());
+  }, [dispatch]);
 
-  const getOperation = () => {
-    // path payment or swap
-    if (isPathPayment || isSwap) {
-      const destMin = computeDestMinWithSlippage(
-        allowedSlippage,
-        destinationAmount,
-      );
-      return StellarSdk.Operation.pathPaymentStrictSend({
-        sendAsset: sourceAsset,
-        sendAmount: amount,
-        destination: isSwap ? publicKey : destination,
+  // checked tags for blocked accounts
+  useEffect(() => {
+    if (isValidatingSafeAssetsEnabled) {
+      const operation = getOperation(
+        sourceAsset,
         destAsset,
-        destMin: destMin.toFixed(7),
-        path: path.map((p) => getAssetFromCanonical(p)),
-      });
-    }
-    // create account if unfunded and sending xlm
-    if (
-      !destinationBalances.isFunded &&
-      asset === StellarSdk.Asset.native().toString()
-    ) {
-      return StellarSdk.Operation.createAccount({
+        amount,
+        destinationAmount,
         destination,
-        startingBalance: amount,
+        allowedSlippage,
+        path,
+        isPathPayment,
+        isSwap,
+        destinationBalances.isFunded!,
+        publicKey,
+      );
+
+      blockedAccounts.forEach(({ address, tags }) => {
+        if (address === operation.destination) {
```

Comment:
Hm, I'm actually having an issue getting the warning to show. It looks like `operation.destination` is undefined 🤔 
<img width="1597" alt="Screenshot 2023-04-07 at 1 29 20 PM" src="https://user-images.githubusercontent.com/6789586/230651714-d00ddb9d-6e99-4df3-8353-08a3503606c8.png">

Reply:
whoops, yeah this is just supposed to be `destination` here, not sure what I was thinking. Thanks for catching it, fixed in 38b4bb1df61ce09bb498335baae79544404581f8